sktime/forecasting/tests/test_naive.py

```diff
@@ -213,7 +213,7 @@ def test_strategy_mean_and_last_seasonal_additional_combinations(
     # For selected cases, remove a redundant data point by making it NaN
     if window_length > sp:
         # create a trailing NaN value in the training set
-        data[window_length - 1] = np.nan
+        data.iloc[window_length - 1] = np.nan
```

Comment:
May be `iat` is more appropritate?

Reply:
do both not do the same?
I just followed the deprec instructions 1:1.

Reply:
From https://pandas.pydata.org/docs/reference/api/pandas.Series.iat.html

> Similar to iloc, in that both provide integer-based lookups. Use iat if you only need to get or set a single value in a DataFrame or Series.